src/recipes.jl

```diff
@@ -5,14 +5,14 @@
 
 dateformatter(dt) = string(Date(Dates.UTD(dt)))
 datetimeformatter(dt) = string(DateTime(Dates.UTM(dt)))
-timeformatter(t) = string(Dates.Time(Dates.Nanosecond(t)))
+timeformatter(t) = string(Dates.Time(Dates.Nanosecond(round(t))))
 
 @recipe f(::Type{Date}, dt::Date) = (dt -> Dates.value(dt), dateformatter)
 @recipe f(::Type{DateTime}, dt::DateTime) =
     (dt -> Dates.value(dt), datetimeformatter)
 @recipe f(::Type{Dates.Time}, t::Dates.Time) = (t -> Dates.value(t), timeformatter)
 @recipe f(::Type{P}, t::P) where {P<:Dates.Period} =
-    (t -> Dates.value(t), t -> string(P(t)))
+    (t -> Dates.value(t), t -> string(P(round(t))))
```

Comment:
If the above comment makes sense, this should also be:
```suggestion
    (t -> Dates.value(t), t -> string(P(round(Int, t))))